Meadowlark-js/backends/meadowlark-mongodb-backend/.npmrc

```
@@ -0,0 +1,3 @@
+registry=https://pkgs.dev.azure.com/ed-fi-alliance/Ed-Fi-Alliance-OSS/_packaging/EdFi/npm/registry/ 
```

With this npmrc file, what happens when you run `npm ci`? When we first started using Azure Artifacts, it did not support upstream feeds. In other words, you could only download _our_ packages from it, not other people's. But I saw something recently that made me think they may have changed that.

If it _does_ fail, then you'll need to change this to use scopes. I think its like this:

```suggestion
@edfi:registry=https://pkgs.dev.azure.com/ed-fi-alliance/Ed-Fi-Alliance-OSS/_packaging/EdFi/npm/registry/ 
```